tests/test_data/small_fake_package_a/setup.py

```diff
@@ -0,0 +1,5 @@
+from setuptools import setup
+
+setup(
+    name="small_fake_with_deps_a", version=0.1, install_requires=["small-fake-a==0.1"]
```

Comment:
The dir name should have the same name as the package name. I know the other dirs do not follow this rule, but I think we should fix it in the following PR.

Reply:
Apologize for the confusion, I meant to change `tests/test_data/small_fake_with_deps/setup.py` to `tests/test_data/small_fake_with_deps_a/setup.py`

Reply:
No problem. I will continue to change the other package name in other PR after this one is merged.

Reply:
Nice! 👍